app/src/sharedTest/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentTest.kt

```diff
@@ -418,6 +423,229 @@ class PlatformParametersFragmentTest {
     }
   }
 
+  @Test
+  fun testPlatformParametersFragment_boolParam_onlyOverriddenValue_returnsOverriddenBoolValue() {
+    executeInPreviousAppInstance { component ->
+      addTestBooleanOverriddenPlatformParameterToDatabase(component, true)
+      component.getTestCoroutineDispatchers().runCurrent()
+    }
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+
+      val splashScreenWelcomeMsgParameter = getEphemeralPlatformParameters()[0]
+      verifyPlatformParameterState(
+        position = 0,
+        expectedValue = splashScreenWelcomeMsgParameter.currentValue
```

Comment:
Ditto as per last comment -- assert against expected values. 
Also, in the above test case.

```suggestion
        expectedValue = PlatformParameterValue.newBuilder()
            .setBoolean(true)
            .build()
```

Reply:
updated